demos/TicTacToe/Classes/TicTacToeLayer.cpp

```diff
@@ -365,6 +382,47 @@ TicTacToeLayer::TicTacToeLayer(string game_uuid) {
   board_sprite->setPosition(0, 0);
   board_sprite->setAnchorPoint(Vec2(0.0, 0.0));
 
+  leave_button_sprite = Sprite::create(kLeaveButtonFileName);
+  if (!leave_button_sprite) {
+    log("kLeaveButtonSprite: %s file not found.", kLeaveButtonFileName);
+    exit(true);
+  }
+  leave_button_sprite->setPosition(450, 585);
+  leave_button_sprite->setAnchorPoint(Vec2(0.0, 0.0));
+  leave_button_sprite->setScale(.35);
+
+  // Create a button listener to handle the touch event.
+  auto leave_button_sprite_touch_listener =
+      EventListenerTouchOneByOne::create();
+  // Setting the onTouchBegan event up to a lambda will swap scenes and modify
+  // total_players
+  leave_button_sprite_touch_listener->onTouchBegan =
+      [this](Touch* touch, Event* event) -> bool {
+    auto bounds = event->getCurrentTarget()->getBoundingBox();
+    auto point = touch->getLocation();
+    // Replaces the scene with a new TicTacToe scene if the touched point is
+    // within the bounds of the button.
+    if (bounds.containsPoint(point)) {
+      // Update the game_outcome to reflect if the you rage quit or left
+      // pre-match.
+      if (remaining_tiles.size() == kNumberOfTiles)
```

Comment:
I don't think the style guide explicitly requires it here, but I prefer to have braces around all my if/else statements.  Unbraced if bodies make me nervous.

Reply:
789ee14 should address this.